### PR TITLE
Support all HTTP methods for local JSON responses

### PR DESCRIPTION
**Pull request by ltazmin**
This PR updates the highwind app to serve all HTTP methods. It also adds a safeguard to prevent the given production URL from being called for non-GET requests.

### CODE REVIEW

**Comment by ltazmin** [comment]
@cribbles Let me know if this makes sense!

File: spec/mock_api_spec.js [hunk 1]
```diff
@@ -69,61 +69,88 @@ describe('start()', function() {
...
+                  if (err) {
+                    done();
+                  }
+                }));
+            });
           });
         });
 
-        describe('When the saveFixtures setting is set to false', function() {
+        describe('When the request method is not GET', function() {
```
Comment (ltazmin) [inline spec/mock_api_spec.js]: The diff is not great, but this is the added test essentially.

**Comment by cribbles** [comment]
LGTM 👍

**Comment by ltazmin** [comment]
Thanks @cribbles!!

**Pull request merged**